react/src/Datasets/components/DatasetTable.tsx

```diff
@@ -167,6 +178,7 @@ export default function DatasetTable() {
                 field: "dataset_type",
                 lookup: datasetTypes,
                 editable: (columnDef, row) => !row.analyses.length,
+                render: RenderDatasetType,
```

Comment:
I think you can inline
```suggestion
                render: rowData => (
                    <TooltipDatasetType dataset_type={rowData.dataset_type}>
                        <div>{rowData.dataset_type}</div>
                    </TooltipDatasetType>
                ),
```
here as it is already in a `useMemo` and do without the `useCallback`.